src/index.js

```diff
@@ -8,13 +8,15 @@ import { Reducers } from './reducers';
 import { Actions } from './actions';
 
 import { applyGridConfig } from './constants/GridConstants';
+import * as ActionTypes from './constants/ActionTypes';
```

Comment:
Would you mind removing rootReducer as one of the exports? It was added a loooong time ago, and it's no longer necessary. Thanks!